node/resolvers/Mutations/Organizations.ts

```diff
@@ -491,117 +644,20 @@ const Organizations = {
       throw new GraphQLError('Organization request already processed')
     }
 
-    const { email, firstName, lastName } = organizationRequest.b2bCustomerAdmin
+    const { email, firstName } = organizationRequest.b2bCustomerAdmin
```

Comment:
Why did we remove the `lastName` here?

Reply:
I don't know why they decomposed the fields but regrouped again [here](https://github.com/vtex-apps/b2b-organizations-graphql/pull/131/files/1aafb503aee96310bdae50d2795290d6486a3a88#diff-ac25af63931d8c82c74361072a12cf08059bdb5a75d58f7fc1586f9d9cb91ef8L520).

Now I am using the method `createOrganizationAndCostCenterWithAdminUser` and use the same [here](https://github.com/vtex-apps/b2b-organizations-graphql/pull/131/files/1aafb503aee96310bdae50d2795290d6486a3a88#diff-ac25af63931d8c82c74361072a12cf08059bdb5a75d58f7fc1586f9d9cb91ef8R138).